primitives/chain-rococo/src/lib.rs

```diff
@@ -42,8 +43,26 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 };
 
 #[derive(parity_scale_codec::Encode, parity_scale_codec::Decode, Debug, PartialEq, Eq, Clone)]
+#[allow(non_camel_case_types)]
 pub enum Call {
```

Comment:
We should have a doc comment here with what our goals are for this enum for future readers.

E.g this should be SCALE compatible with what's in the runtime without necessarily using the runtime

Reply:
Added comments and test.